conda_environments/testing_py35.yml

```diff
@@ -10,5 +10,5 @@ dependencies:
 - libgdf=0.1.0a3.*
 - libgdf_cffi=0.1.0a3.*
 - numba>=0.40.0dev
-- pandas=0.20.3
+- pandas>=0.23.4
```

Comment:
We should pin this specifically to 0.23.4 for now because we can't guarantee we won't break with future Pandas versions.